### PR TITLE
Require valgrind-3.17

### PR DESCRIPTION
**Pull request by baldersheim**
@aressem or @toregge PR

### CODE REVIEW

File: dist/vespa.spec [hunk 1]
```diff
@@ -232,7 +232,7 @@ Requires: vespa-protobuf = 3.7.0-5.el7
 %endif
 %if ! 0%{?amzn2}
 Requires: vespa-telegraf >= 1.1.1-1
-Requires: vespa-valgrind >= 3.16.0-1
+Requires: vespa-valgrind >= 3.17.0-1
```
Comment (aressem) [inline dist/vespa.spec:235]: This package does not exist ?

**Comment by aressem** [comment]
Requires packages to be available. Version number bumped in https://github.com/vespa-engine/vespa-3rdparty-deps/pull/61

File: dist/vespa.spec (hunk 1 above)
Comment (baldersheim) [reply dist/vespa.spec:235]: When will it be available ?

Comment (baldersheim) [reply dist/vespa.spec:235]: .. After merge of https://github.com/vespa-engine/vespa-3rdparty-deps/pull/61

Comment (aressem) [reply dist/vespa.spec:235]: This is currently a manual process. Sent you the details.

**Review by aressem — APPROVED** [review]

**Pull request merged**